azure-mgmt-eventhub/src/test/java/com/microsoft/azure/management/eventhub/EventHubTests.java

```diff
@@ -88,10 +88,10 @@ public void canManageEventHubNamespaceBasicSettings() {
 
         EventHubNamespace namespace3 = eventHubManager.namespaces()
                 .define(namespaceName3)
-                .withRegion(REGION)
-                .withExistingResourceGroup(RG_NAME)
-                .withSku(EventHubNamespaceSkuType.BASIC)
-                .create();
+                    .withRegion(REGION)
+                    .withExistingResourceGroup(RG_NAME)
+                    .withSku(EventHubNamespaceSkuType.BASIC)
```

Comment:
In SQL for databases and elastic pools I simplified this further: .withBasicSku() or .withStandardSku() etc.
It makes sense if the event hub SKU selection is small enough. 
Is it possible to combine the SKU selection with the throughput unit setting? If yes, you could combine the two like:
.withStandardSku(int capacity)... Just an idea :-)

Reply:
you could overload the .withBasicSku() to take one (the upper limit I will guess) or both as parameters. In SqlDatabases I have a similar pattern where user can specify the edition, the service objective and the max storage capacity.

Reply:
Yes agree there is a different ways to express it but as I mentioned my intention was to make it clear that these two totally differnt properties in the payload are logically related, hence used matchng names. In addition this is how eventhub wanted to express if we refer their docs.